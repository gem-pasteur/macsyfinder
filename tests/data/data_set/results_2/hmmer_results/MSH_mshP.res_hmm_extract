# gene: MSH_mshP extract from results_2/hmmer_results/MSH_mshP.search_hmm.out hmm output
# profile length= 59
# i_evalue threshold= 0.001
# coverage threshold= 0.500
# hit_id replicon_name position_hit hit_sequence_length gene_name gene_system i_eval score profile_coverage sequence_coverage begin end
