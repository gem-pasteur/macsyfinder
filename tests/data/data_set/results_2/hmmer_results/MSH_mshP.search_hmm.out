# hmmsearch :: search profile(s) against a sequence database
# HMMER 3.1b2 (February 2015); http://hmmer.org/
# Copyright (C) 2015 Howard Hughes Medical Institute.
# Freely distributed under the GNU General Public License (GPLv3).
# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
# query HMM file:                  tests/data/data_set/models/set_1/profiles/MSH_mshP.hmm
# target sequence database:        tests/data/base/one_replicon.fasta
# output directed to file:         results_2/hmmer_results/MSH_mshP.search_hmm.out
# sequence reporting threshold:    E-value <= 0.1
# number of worker threads:        0
# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -

Query:       MSH_mshP  [M=59]
Scores for complete sequences (score includes all domains):
   --- full sequence ---   --- best 1 domain ---    -#dom-
    E-value  score  bias    E-value  score  bias    exp  N  Sequence             Description
    ------- ------ -----    ------- ------ -----   ---- --  --------             -----------
     0.0012   15.4   0.2     0.0028   14.3   0.2    1.6  1  GCF_000006845_004220  


Domain annotation for each sequence (and alignments):
>> GCF_000006845_004220  
   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc
 ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----
   1 !   14.3   0.2   1.5e-06    0.0028       1      23 [.      17      39 ..      17      55 .. 0.79

  Alignments for each domain:
  == domain 1  score: 14.3 bits;  conditional E-value: 1.5e-06
              MSH_mshP  1 rGsaLiivlFvIvvmallAaall 23
                          rGsaL ivl v +v+a+l  ++ 
  GCF_000006845_004220 17 RGSALFIVLMVMIVVAFLVVTAA 39
                          7***********99999965544 PP



Internal pipeline statistics summary:
-------------------------------------
Query model(s):                              1  (59 nodes)
Target sequences:                         1886  (549846 residues searched)
Passed MSV filter:                        81  (0.042948); expected 37.7 (0.02)
Passed bias filter:                       44  (0.0233298); expected 37.7 (0.02)
Passed Vit filter:                         5  (0.00265111); expected 1.9 (0.001)
Passed Fwd filter:                         1  (0.000530223); expected 0.0 (1e-05)
Initial search space (Z):               1886  [actual number of targets]
Domain search space  (domZ):               1  [number of targets reported over threshold]
# CPU time: 0.00u 0.00s 00:00:00.00 Elapsed: 00:00:00.00
# Mc/sec: inf
//
[ok]
